src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -17,27 +19,27 @@ public Figure getRandomFigure() {
     }
 
     private Figure createCircle() {
-        return new Circle((int) (Math.random() * RANDOM_VALUE + MIN_VALUE));
+        return new Circle(RANDOM.nextInt(RANDOM_VALUE));
     }
 
     private Figure createIsoscelesTrapezoid() {
         return new IsoscelesTrapezoid(
-                (int) (Math.random() * RANDOM_VALUE + MIN_VALUE),
-                (int) (Math.random() * RANDOM_VALUE + MIN_VALUE),
-                (int) (Math.random() * RANDOM_VALUE + MIN_VALUE));
+                RANDOM.nextInt(RANDOM_VALUE),
+                RANDOM.nextInt(RANDOM_VALUE),
+                RANDOM.nextInt(RANDOM_VALUE));
```

Comment:
what about this comment? https://github.com/mate-academy/jv-oop-advanced/pull/196#discussion_r691978997